mkosi/__init__.py

```diff
@@ -779,13 +779,7 @@ def install_unified_kernel(state: MkosiState, roothash: Optional[str]) -> None:
                 import pefile  # type: ignore
                 pe = pefile.PE(boot_binary, fast_load=True)
                 linux = {s.Name.decode().strip("\0"): s for s in pe.sections}[".linux"]
-                run(["dd",
-                     f"if={boot_binary}",
-                     f"of={state.staging / state.config.output_split_kernel}",
-                     f"skip={linux.PointerToRawData}",
-                     # Get the actual size using Misc_VirtualSize instead of the aligned size from SizeOfRawData.
-                     f"count={linux.Misc_VirtualSize}",
-                     "iflag=skip_bytes,count_bytes"])
+                (state.root / state.config.output_split_kernel).write_bytes(linux.get_data(length=linux.Misc_VirtualSize))
```

Comment:
This should do:

```suggestion
                (state.root / state.config.output_split_kernel).write_bytes(linux.get_data(ignore_padding=True))
```

Reply:
I had that first, it's too new, ignore_padding() is not available in Jammy's version of pefile